### PR TITLE
Add functionality to export users in CSV format

### LINKED ISSUE #443: UI confusing in http://localhost:4502/mnt/overlay/netcentric/actool/content/overview.html/
The newly added Touch UI is confusing with respect to the order of buttons. 
Screenshot 2020-03-27 at 11 28 15

The button "Download Dump" is on the same line as "Configuration Root Path" although it has nothing to do with it.

I would propose to place the "Download Dump" in a dedicated row, so that it is clear that this action cannot be parameterized! Also it would be good to explain whether the dump is by authorizable/path (and what that means).

The same applies to the WebConsole at http://localhost:4502/system/console/actool

### LINKED ISSUE #703: Export Users in CSV format (compatible with Admin Console)
When you migrate from AEM Classic to Cloud you often want to leverage Adobe IMS for maintaining users and their direct group memberships. To ease this migration the JMX Console as well as the Web UI (/mnt/overlay/netcentric/actool/content/overview.html/actool) should provide a way to create a compliant CSV file (https://helpx.adobe.com/enterprise/using/bulk-upload-users.html#csv-format) which can afterwards be imported in the Admin Console.

### PR DESCRIPTION
**Pull request by kwin**
This format is compatible with
https://helpx.adobe.com/enterprise/using/bulk-upload-users.html#csv-format

Fix Web Console deployment with sling-m-p

This closes #703
This closes #443

### CODE REVIEW

File: accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java [hunk 1]
```diff
@@ -44,11 +61,15 @@ public class AcToolUiService {
...
 
-    static final String PATH_SEGMENT_DUMP = "dump.yaml";
+    static final String SUFFIX_DUMP_YAML = "dump.yaml";
+    static final String SUFFIX_USERS_CSV = "users.csv";
 
     @Reference(policyOption = ReferencePolicyOption.GREEDY)
     private ConfigDumpService dumpService;
 
+    @Reference(policyOption = ReferencePolicyOption.GREEDY)
+    private UserProcessor userExporter;
```
Comment (ghenzler) [inline accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java]: What about calling the interface also`UserExpoerter`? Also would call the package of that service `userexporter`

Comment (kwin) [reply accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java]: This service does not export (in the sense of serializing or converting) something. The actual export is done within this service.

Comment (kwin) [reply accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java]: Just rename variable to userProcessor.

File: accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java [hunk 2]
```diff
@@ -93,18 +141,60 @@ protected void doPost(final HttpServletRequest req, final HttpServletResponse re
         }
     }
 
+    
+    private boolean mayApplyConfig(HttpServletRequest req) {
```
Comment (kwin) [inline accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/ui/AcToolUiService.java]: hasAccessToFelixWebConsole()

**Pull request merged**